tests/src/Functional/GroupUpdateTest.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Functional\GroupUpdateTest.
+ */
+
+namespace Drupal\Tests\og\Functional;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\node\Entity\Node;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+use Drupal\og\OgAccess;
+use Drupal\og\OgMembershipInterface;
+use Drupal\simpletest\ContentTypeCreationTrait;
+use Drupal\simpletest\NodeCreationTrait;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the special permission 'update group'.
+ *
+ * @group og
+ */
+class GroupUpdateTest extends BrowserTestBase {
```

Comment:
I will extend the OgEntityAccess test as part of this PR then as well. If this is too much, I can remove it if you think the other part is ok.